presentation/testing/src/presentation-testing/ContentBuilder.ts

```diff
@@ -3,17 +3,18 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
+import { PropertyRecord } from "@itwin/appui-abstract";
```

Comment:
this needs to be moved next to other imports

Reply:
moved.